File: Python/subprogramas/ordenar_vector_ascendente.py

def ordenar_vector_ascendente(vector):
    for i in range(len(vector)):
        for j in range(i + 1, len(vector)):
            if vector[i] > vector[j]:
                vector[i], vector[j] = vector[j], vector[i]

# Example usage
if __name__ == "__main__":
    vector = [3, 1, 4, 1, 5, 9, 2, 6, 5]
    ordenar_vector_ascendente(vector)
    print("Vector ordenado:", vector)

File: C++/subprogramas/ordenar_vector_ascendente.cpp

#include <iostream>
#include <vector>

using namespace std;

void ordenar_vector_ascendente(vector<int>& vector) {
    for (size_t i = 0; i < vector.size(); ++i) {
        for (size_t j = i + 1; j < vector.size(); ++j) {
            if (vector[i] > vector[j]) {
                swap(vector[i], vector[j]);
            }
        }
    }
}

int main() {
    vector<int> vector = {3, 1, 4, 1, 5, 9, 2, 6, 5};
    ordenar_vector_ascendente(vector);
    cout << "Vector ordenado: ";
    for (int val : vector) {
        cout << val << " ";
    }
    cout << endl;
    return 0;
}